### PR TITLE
[MTM-52741] Mention which type has c8y_Connection in availability mon…

### PR DESCRIPTION
**Pull request by Michal-Bak**
…itoring.

### CODE REVIEW

File: content/reference/device-integrator-library-bundle/device-availability.md [hunk 1]
```diff
@@ -81,11 +81,11 @@ Keep in mind that after updating the last message it may take some minutes until
 
 A monitored device has one of the following statuses for `c8y_Connection`:
 
-|Name| Description                                                                            |
-|:---|:---------------------------------------------------------------------------------------|
-|CONNECTED| A device push connection is established.                                               |
-|DISCONNECTED| `responseInterval` is larger than 0 and the device is neither AVAILABLE nor CONNECTED. |
-|MAINTENANCE| `responseInterval` is smaller or equal to 0; the device is under maintenance.          |
+| Status (String) | Description                                                                            |
```
Comment (rssag) [inline content/reference/device-integrator-library-bundle/device-availability.md:84]:
```suggestion
| Status (string) | Description                                                                            |
```

**Review by rssag — COMMENTED** [review]
a small suggestion + approval

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
what are the target versions?

**Comment by Michal-Bak** [comment]
> what are the target versions?

it is minor clarification, it can go only for the newest version.

**Review by kowalczykmike — APPROVED** [review]

**Pull request merged**